ecosystem/sep-0030.md

```diff
@@ -0,0 +1,479 @@
+accounts threshold to be able to successfully submit a transaction.
+
+The client can use this process to sign a transaction with the [Set Options
+operation] to add a new signer to the account to regain control of an account
+of which they have lost the key, or to gain control of an account shared with
+them.
+
+A client can also use the endpoints in the specification to:
+- Discover registered accounts that it has access to.
+- Update the identities associated with an account.
+- Delete accounts.
+
+## Specification
+
+### Authentication
+
+All endpoints require authentication in the form of a [JWT] token that is passed
+in the `Authorization` header. The [JWT] can be issued by two types of issuers.
+Some endpoints require a specific provider while others should function with
+both. The two issuers are:
+
+- `SEP-10`: A [SEP-10] Web Auth server for proving high threshold control of an
+account. The [SEP-10] server should be configured to only issue [JWT]s if signers
+prove they meet the high threshold on the account.
+- `External`: Defined by the implementer. Proves control of a phone number,
+email address, or other identity.
+
+### Content Type
+
+All endpoints accept in requests the following `Content-Type`s:
+- `application/x-www-form-urlencoded`
+- `application/json`
+
+All endpoints respond with content type:
+- `application/json`
+
+### Errors
+
+If an error occurs when calling any endpoint, an appropriate HTTP status code
+will be returned along with an error response.
+
+#### Status Code
+
+Common HTTP status codes may be returned for a server. In particular the following are expected:
+
+Status Code | Name | Reason
+-----|------|------------
+`400` | Bad Request | The request is invalid in anyway.
+`401` | Unauthorized | No `Authorization` header has been provided or the contents of the header are not accepted as valid.
+`404` | Not Found | The path is not recognized by the server, or the address does not have an account, or the authenticated user does not have permission to access the account.
+`409` | Conflict | The account is already registered, when attempting to register an account that is already registered.
+
+#### Response
+
+##### Fields
+
+Name | Type | Description
+-----|------|------------
+`error` | string | A description of the error.
+
+##### Example
+
+```json
+{
+   "error": "..."
+}
+```
+
+### Common Fields
+
+A set of common fields are used to describe an account in most endpoints:
+
+Name | Type | Description
+-----|------|------------
+`address` | string | The Stellar account ID or address of the account.
+`identities.has_owner` | boolean | True if the account has alternative identities set for the owner identity.
+`identities.has_other` | boolean | True if the account has alternative identities set for the other identity.
+`identities.*.account` | string | The Stellar account ID of an another account that if authenticated will be given full permissions of this account.
```

Comment:
I was thinking of replacing the `*` with `owner|other`. It made the table wide though. I was also thinking that reading all these fields upfront is not necessarily very useful because the reader hasn't seen the context they are used in yet. What do you think if I move them to under the endpoints?

Reply:
I've restructured this section, check it out. 904b04c

Reply:
Looks much better, thanks!